Doxyfile.in

```diff
@@ -618,7 +618,7 @@ CITE_BIB_FILES         =
 # The QUIET tag can be used to turn on/off the messages that are generated
 # by doxygen. Possible values are YES and NO. If left blank NO is used.
 
-QUIET                  = NO
+QUIET                  = YES
```

Comment:
I think we should generally be noisy, particularly for things like Travis, although I understand why you may have turned it off during testing.